sdk/eventhub/azure-eventhub/migration_guide.md

```diff
@@ -1,266 +1,178 @@
+## Important changes
 
-### Sending events
-The process of building event batches is more transparent with `send_batch` of V5.
+### Client hierarchy
+In the interest of simplifying the API surface, we've made two distinct clients, rather than having a single `EventHubClient`:
+* `EventHubProducerClient` for sending events. 
+* `EventHubConsumerClient` for receiving events. 
 
-| In v1 | Equivalent in v5 | Sample |
-|---|---|---|
-| `EventHubClientAsync.add_async_sender()` and `AsyncSender.send()`| `EventHubProducerClient.send_batch()`| [send events](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/eventhub/azure-eventhub/samples/async_samples/send_async.py) |
+We've also merged the functionality from `EventProcessorHost` into `EventHubConsumerClient`. 
 
-## Migration samples
+#### Approachability
+By having a single entry point for sending, the `EventHubProducerClient` helps with the discoverability of the API
+as you can explore all available features for sending events through methods from a single client, as opposed to searching
+through documentation or exploring namespace for the types that you can instantiate.
 
-* [Receiving events](#migrating-code-from-eventhubclient-and-asyncreceiver-to-eventhubconsumerclient-for-receiving-events)
-* [Sending events](#migrating-code-from-eventhubclient-and-asyncsender-to-eventhubproducerclient-for-sending-events)
-* [Receiving events with checkpointing](#migrating-code-from-eventprocessorhost-to-eventhubconsumerclient-for-receiving-events)
+Similarly, by having a single entry point for receiving of any type (from single partition, all partitions, or with load balancing and checkpointing features) within Event Hubs, the `EventHubConsumerClient` helps with the discoverability of the API as you can explore all available features for receiving events through methods from a single client, as opposed to searching
+through documentation or exploring namespace for the types that you can instantiate.
 
-### Migrating code from `EventHubClient` and `AsyncReceiver` to `EventHubConsumerClient` for receiving events
+#### Consistency
+We now have methods with similar names, signature and location for sending and receiving.
+This provides consistency and predictability on the various features of the library.
 
-In V1, `AsyncReceiver.receive()` returns a list of EventData.
+### Client constructors
 
-In V5, EventHubConsumerClient.receive() calls user callback on_event to process events.
+- While we continue to support connection strings when constructing a client, the main difference is when using Azure Active Directory.
+We now use the new [`azure-identity`](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/identity/azure-identity/README.md) library
+to share a single authentication solution between clients of different Azure services.
 
-For example, this code which keeps receiving from a partition in V1:
+In v1:
```

Comment:
i believe there's some value in providing an inline sample wherever available for each of the v5 versions.

Reply:
I think a table would have worked. However, since I expanded the constructor samples a little, an inline sample seems like it would be a little messy? Would be up for discussing after I update with the eph sample as well for receiving.